docs/api/apiv3/paths/storage_files_prepare_upload.yml

```diff
@@ -64,8 +64,17 @@ post:
             errorIdentifier: urn:openproject-org:api:v3:errors:NotFound
             message: The requested resource could not be found.
       description: |-
-        Returned in either of those cases:
-        - if the storage does not exist or the client does not have sufficient permissions to see it
+        Returned the storage does not exist or the client does not have sufficient permissions to see it.
 
-          **Required permission:** view file links
-        - if the document root file identification does not exist on the storage
+        **Required permission:** view file links
+    '500':
+      content:
+        application/hal+json:
+          schema:
+            $ref: '../components/schemas/error_response.yml'
+          example:
+            _type: Error
+            errorIdentifier: urn:openproject-org:api:v3:errors:OutboundRequestFailure
```

Comment:
What about `OutboundRequest:NotFound`. Now it looks more like a general (🫡) failure. IMO we can be specific here.

Reply:
That's fine for me.